pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm[toml]>=6.2", "numpy"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "uvtools"
+description = "Tools useful for the handling, visualization, and analysis of interferometric data."
+authors = [
+    {name="HERA Team"},
+]
+readme = "README.md"
+requires-python = ">=3.8"
+license = {text = "MIT"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+]
+dependencies = [
+    "numpy",
+    "six",
+    "scipy",
+    "aipy",
+    "pyuvdata",
+    "astropy>5.1"
+]
+dynamic = [
+    "version",
+]
+
+
+[project.optional-dependencies]
+aipy = ["aipy>=3.0rc2"]
```

Comment:
should `aipy` be listed under dependencies _and_ optional?

Reply:
Good call. It's optional.